www/webapp/.eslintrc.js

```diff
@@ -24,17 +28,11 @@ module.exports = {
     'vue/no-deprecated-html-element-is': 'warn', // Preparation for vue3
   },
   ignorePatterns: ['**/src/modules/**/*'],
-  parserOptions: {
-    parser: '@babel/eslint-parser'
-  },
   overrides: [
     {
       files: [
-        '**/__tests__/*.{j,t}s?(x)'
+        'src/**/*.{vue,js,jsx,cjs,mjs,ts,tsx,cts,mts}'
```

Comment:
Does this actually override anything? (This override does not have a `rules` key.)

Reply:
Good catch. Seems to be a leftover. If I remember correctly the override "filtered" the input by the all files of "eslint src/". Override seems useless with the current version.